socobo-inventory-overview.html

```diff
@@ -42,20 +42,23 @@
     </style>
     <div>
       <paper-material vertical layout class="inventory" elevation="1">
-        <paper-toolbar class="searchBox">
-          <paper-checkbox class="checkBoxAll" id="inventoryCheckBoxAll"></paper-checkbox>
-          <paper-input class="search-input" id="inventorySearch" label="Suche"></paper-input>
-        </paper-toolbar>
+        <socobo-element-search 
+          id="searchHeader" 
+          on-check-all-items="_handleCheckAll"
+          on-uncheck-all-items="_handleUncheckAll"
+          on-changed-search-query="_searchQueryChanged">
```

Comment:
me? can't remember. but if you look into my documentation. you see all possible things!